lint/stylelint.bzl

```diff
@@ -0,0 +1,208 @@
+"""Configures [Stylelint](https://stylelint.io/) to run as a Bazel aspect
+
+First, all CSS sources must be the srcs of some Bazel rule.
+You can use a `filegroup` with `lint-with-stylelint` in the `tags`:
+
+```python
+filegroup(
+    name = "css",
+    srcs = glob(["*.css"]),
+    tags = ["lint-with-stylelint"],
+)
+```
+
+See the `filegroup_tags` and `rule_kinds` attributes below to customize this behavior.
+
+## Usage
+
+Add `stylelint` as a `devDependency` in your `package.json`, and declare a binary target for Bazel to execute it.
+
+For example in `tools/lint/BUILD.bazel`:
+
+```starlark
+load("@npm//:stylelint/package_json.bzl", stylelint_bin = "bin")
+stylelint_bin.stylelint_binary(name = "stylelint")
+```
+
+Then declare the linter aspect, typically in `tools/lint/linters.bzl`:
+
+```starlark
+load("@aspect_rules_lint//lint:stylelint.bzl", "lint_stylelint_aspect")
+stylelint = lint_stylelint_aspect(
```

Comment:
I noticed this was `... = vale_aspect(` a second ago... looking at the READMEs it looks like there are number of places that are missing the `lint_` prefix in the docstrings where it should be `... = lint_foo_aspect`